exercises/plot_dets_filesize.py

```diff
@@ -0,0 +1,94 @@
+from databroker import Broker
+import pandas as pd
+
+import os
+
+import datetime
+import time
+from time import mktime
+
+from eiger_io.fs_handler import EigerHandler
+from databroker.assets.handlers import AreaDetectorTiffHandler
+
+
+def find_keys(hdrs, db):
+    '''
+        This function searches for keys that are stored via filestore in a
+        database, and gathers the SPEC id's from them.
+    '''
+    FILESTORE_KEY = "FILESTORE:"
+    keys_dict = dict()
```

Comment:
you can use `defaultdict`:

```py
from collections import defaultdict
keys_dict = defaultdict(lambda : int(0))
```

then further down you can remove the if statements (see comment below).
it's a nice trick to reduce lines of code :)